fractal_server/app/schemas/user.py

```diff
@@ -93,6 +96,7 @@ class UserCreate(schemas.BaseUserCreate):
     slurm_user: Optional[str]
     cache_dir: Optional[str]
     username: Optional[str]
+    slurm_accounts: Optional[list[str]] = []
```

Comment:
I don't think this is correct.
1. `Optional` means that the type can be `None`, but this is not a valid use case.
2. I think using mutable objects as default is not recommended for class attributes either (apart from being plain wrong for function arguments). Isn't there some better way, possibly based on `Field`?